admin/class-wp-security-bp-admin.php

```diff
@@ -78,15 +70,8 @@ class WP_Security_BP_Admin {
 	public function __construct( $plugin_name, $version ) {
 
 		$this->plugin_name = $plugin_name;
-		$this->version = $version;
-		$this->admin_url = admin_url( 'options-general.php?page=' . $this->plugin_name );
-		/* $this->json = array(
-			'status'   		=> 'fail',
-			'short_desc'	=> '',	
-			'message'  		=> '',
-			'button'   		=> false,
-			'uri'      		=> '',
-		); */
+		$this->version     = $version;
+		$this->admin_url   = admin_url( 'options-general.php?page=' . $this->plugin_name );
 
```

Comment:
![equals-not-aligned](https://user-images.githubusercontent.com/43264089/53843885-318cac80-3fa4-11e9-9352-2a14c833405d.JPG)